### PR TITLE
make package boring, rename package to scoped, add async methods

### PR DESCRIPTION
**Pull request by sveisvei**
(Messed up commits abit by doing a wrong reset before push.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +75,19 @@ Supported arguments are:
  - `options.path` - String - path to where files whould be / are persisted.
 
 
+### get(fileName: string): Promise<string>
+
+Async method for reading a file from storage
+
+### set(fileName: string, fileContent: string): Promise<void>
+
+Async method for writing file to storage
```
Comment (digitalsadhu) [inline README.md]: s/file/a file

File: README.md [hunk 2]
```diff
@@ -60,6 +75,19 @@ Supported arguments are:
...
+
+### has(fileName: string): Promise<Boolean>
+
+Async method for checking if file exist in storage
```
Comment (digitalsadhu) [inline README.md]: s/file exist/a file exists

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!-- LICENSEFILE/ -->
+
+<h1>License</h1>
```
Comment (SimenB) [inline LICENSE.md]: why markup and not the standard license?

File: lib/sink.js [hunk 4]
```diff
@@ -47,38 +55,69 @@ class WriteStream extends stream.PassThrough {
...
+    writer(type) {
         assert(type, '"type" is missing');
         return new WriteStream(this.options, type);
     }
 
-    reader (file) {
-        assert(file, '"file" is missing');
-        return new ReadStream(path.join(this.options.path, file));
+    reader(fileName) {
+        assert(fileName, '"fileName" is missing');
```
Comment (digitalsadhu) [inline lib/sink.js]: you could do something like: 
```js
assert(typeof fileName === 'string', `Expected "fileName" to be a string. Instead got ${fileName}`)
```
to catch more cases.

File: LICENSE.md (hunk 3 above)
Comment (sveisvei) [reply LICENSE.md]: cc @digitalsadhu

File: test/sink.test.js [hunk 5]
```diff
@@ -0,0 +1,236 @@
+'use strict';
+
+const stream = require('readable-stream');
+const Sink = require('../');
+const fs = require('fs');
+const os = require('os');
+
+function getFreshSink(key = 'x') {
```
Comment (digitalsadhu) [inline test/sink.test.js]: mmmm dat fresh stink

File: lib/sink.js (hunk 4 above)
Comment (sveisvei) [reply lib/sink.js]: https://github.com/asset-pipe/asset-pipe-sink-fs/commit/7514ea54d60f6c0522d57e2720ffee2bbace7a97

**Pull request merged**